.travis.yml

```diff
@@ -4,6 +4,7 @@ sudo: required
 
 scala:
   - "2.11.8"
+  - "2.12.0"
```

Comment:
Yes. I deleted the custom script. Travis applies the following default script: `sbt ++$TRAVIS_SCALA_VERSION test`. All schould be fine.